src/App.jsx

```diff
@@ -1,29 +1,27 @@
 import './App.sass';
 import React from 'react';
-import { Switch, Route, Redirect } from 'react-router-dom';
+import { Route, Redirect, useLocation } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { ThemeProvider } from '@mui/material/styles';
 import { theme } from './utils';
-import { Home, Login, Internships } from './pages';
+import { Home, Login } from './pages';
 
 function App(props) {
   const { isAuthorized } = props;
+  const currentPath = useLocation().pathname;
```

Comment:
```suggestion
  const { pathname } = useLocation();
```
This way it's more readable